openapi.yaml

```diff
@@ -12021,6 +12021,23 @@ paths:
                 $ref: '#/components/schemas/Tag'
         default:
           $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+        - lang: Shell
+          source: >
+            curl -H "Content-Type: application/json" \
+                -H "Authorization: Bearer $TOKEN" \
+                -X POST -d '{
+                    "label": "example tag",
+                    "linodes": [123,456],
+                    "volumes": [9082,10003]
+                }' \
+                https://api.linode.com/v4/tags
+        - lang: CLI
+          source: >
+            linode-cli tags create \
+              --label 'example tag' \
+              --linodes (123 456) \
```

Comment:
@Dorthu I realized when trying to write this example that I don't know how to represent lists as arguments for the Linode CLI. Using the Bash array syntax here didn't work; I got the following error: 

```
linode-cli tags create \
  --label 'example tag' \
  --linodes (123 456) \
  --volumes (9082 10003)
zsh: unknown file attribute: 1
```

Any tips for where I might be going wrong?

Reply:
You need to specify `--linodes` multiple times; in this case you want: 

```base
linode-cli tags create \
  --label 'example tag' \
  --linodes 123 --linodes 456) \
  --volumes 9082 --volumes 10003
```